### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by rubinnn**
# Things done
Update quickstart of User guide
to be more beginner friendly

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -18,7 +18,21 @@ ClientHub is a **desktop app for managing contacts, optimized for use via a Comm
...
+3. Type `cd Downloads/[FolderName]` and press Enter
+4. Type `java -jar clienthub.jar` and press Enter
+
+**For Windows**
+1. If you have downloaded the jar file into a folder in your `Downloads`
+2. Open PowerShell
+3. Type `cd Downloads/[FolderName]` and press Enter
+4. Type `java -jar clienthub.jar` and press Enter
+
+
```
Comment (jereeemyyyy) [inline docs/UserGuide.md:35]: lgtm!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -508,9 +508,9 @@ testers are expected to do more *exploratory* testing.
 
 * **For Mac**
 
-    1. Download the jar file and copy into an empty folder called `ClientHub` in `Desktop`. 
+    1. Download the jar file and copy into an empty folder called `ClientHub` in `Downloads`. 
     2. Open Terminal 
-    3. Inside the terminal enter: `cd/Desktop/ClientHub`
+    3. Inside the terminal enter: `cd Downloads/ClientHub`
```
Comment (ahmadsyuaib) [inline docs/DeveloperGuide.md:513]: Good catch!

**Review by jereeemyyyy — COMMENTED** [review]
LGTM 🚀

File: docs/UserGuide.md (hunk 1 above)
Comment (ahmadsyuaib) [inline docs/UserGuide.md:35]: Good addition to user guide!

**Review by ahmadsyuaib — APPROVED** [review]

**Pull request merged**